docs/graphql/manual/mutations/insert.rst

```diff
@@ -371,6 +371,91 @@ Insert an object with a JSONB column
       }
     }
 
+Insert an object with an array column
+------------------------------------
+If you need to evaluate an array field you can use ``jsonb`` or ``literal strings``.
+
+**Example 1:** Insert a new ``article`` object with a array ``tags`` column using jsonb:
+
+.. graphiql::
+  :view_only:
+  :query:
+    mutation insert_article_with_tags($tags: jsonb) {
+      insert_article(
+        objects: [
+          {
+            title: "Article 1",
+            content: "Sample article content",
+            author_id: 3,
+            tags: $tags
+          }
+        ]
+      ) {
+        returning {
+          id
+          title
+          tags
+        }
+      }
+    }
+  :response:
+    {
+      "data": {
+        "insert_article": {
+          "affected_rows": 1,
+          "returning": [
+            {
+              "id": 21,
+              "title": "Article 1",
+              "tags": ["tag1", "tag2"]
+            }
+          ]
+        }
+      }
+    }
+  :variables:
+    {
+      "tags": ["tag1", "tag2"]
+    }
+    
+**Example 2:** Insert a new ``article`` passing tags as literal strings:
```

Comment:
```suggestion
**Example 2:** Insert a new ``article`` passing tags as literal strings:

```

Reply:
There's a white space missing which breaks the display of the example:

![image](https://user-images.githubusercontent.com/5722022/69528952-8d062c80-0f6f-11ea-93e5-d35195536bec.png)